### ISSUE TITLE
Feature Request: Add an option to create individual Prefabs from multiple selected entities

### ISSUE THREAD

**Issue by LB-ArturZieba** [o3de/o3de#6961]
**Is your feature request related to a problem? Please describe.**
When multiple entities are selected in the Entity Outliner, and they are right-clicked, there is a "Create Prefab..." option in the dropdown menu. In this case, "Create Prefab..." option creates a single Prefab that contains all selected entities. Currently if a user would like to e.g. have three separate Prefabs made from three different entities, the "Create Prefab..." would have to be performed on each individual entity, which can get time-consuming in certain workflows. 

**Describe the solution you'd like**
An additional "Create Individual Prefabs..." option in the dropdown when selecting multiple entities in the Entity Outliner, which would create a separate prefab for each selected entity.

**Describe alternatives you've considered**
An additional popup dialog when clicking the "Create Prefab..." option with multiple entities selected that asks the user whether to create a single Prefab with all entities, or to separate them to individual Prefabs.

**Additional context**

https://user-images.githubusercontent.com/86952082/149900530-5cc79784-add8-4052-af59-a946c8c2b59c.mp4

**Comment by amzn-leenguy** [comment]
@rainbj is the best person to make the call on this, but I am another UX Designer that can weigh in here from the @o3de/sig-ui-ux . 

I think your suggestion of just adding another menu item below the Create Prefabs... menu option called Create Individual Prefabs... (only appearing if multiple entities are selected) is fine.

@AMZN-daimini I can see if this change was made, some usability testing could answer if it is confusing or not. And asking about 5-8 users would answer that.

**Comment by AMZN-daimini** [comment]
> @AMZN-daimini I can see if this change was made, some usability testing could answer if it is confusing or not. And asking about 5-8 users would answer that.

Agreed. I just don't know how it would work since "Create Prefab..." currently needs to trigger a "Save As..." dialog to get the prefab location. We want to move away from that in the future, but foe now it's a requirement.

**Comment by amzn-leenguy** [comment]
I see your dilemma now @AMZN-daimini . I think launching the Save As dialogs one after another is not a good UX.

For some reason I imagined that creating a prefab wouldn't need to prompt the user to save as, as we could most likely assume the location based off Project Folder (already being defined when the project is created)

Currently Mac OSX handles multiple events with this dialog: 
![Screen Shot 2022-01-20 at 4 53 57 PM](https://user-images.githubusercontent.com/82846749/150446005-1e3b4e1e-40d5-411c-97d0-d1d4d4299c79.png)

Not that I'm saying this is the solution, but I am just brainstorming here.

I wonder if prompting the user with a dialog to say that: "You are attempting to save multiple individual prefabs at once. Do you want to save all these to the same location or different locations?"

[ Same location ] > (prompts Save as Folder workflow)
[ Different locations ] > (prompts multiple Save as File workflows)

[ Cancel ]

I'm just brainstorming. Not sure if it's even a good idea :)

**Comment by AMZN-daimini** [comment]
The long-term plan would be to allow users to just create prefabs in memory, and only prompt for a location to save them in once the user explicitly wants to save them. We still have some technical limitations before we can get there though.

**Comment by amzn-leenguy** [comment]
> The long-term plan would be to allow users to just create prefabs in memory, and only prompt for a location to save them in once the user explicitly wants to save them. We still have some technical limitations before we can get there though.

Okay. Until then, I think we should hold up on this suggestion until we can do that then.

**Comment by monroegm** [comment]
Note for next triage meeting: recommend accepting the request and setting it to blocked.
